app/src/main/java/com/plenart/organizeme/fragments/CreateBoardFragment.kt

```diff
@@ -50,30 +51,32 @@ class CreateBoardFragment : Fragment() {
 
     private fun initListeners() {
         binding.ivBoardImageCreateBoardActivity.setOnClickListener {
-            if(Constants.isReadExternalStorageAllowed(requireActivity())){
+            if (Constants.isReadExternalStorageAllowed(requireActivity())) {
                 Constants.showImageChooser(requireActivity())
```

Comment:
@bagy94 already mentioned this. If class named Constants then there are just constants values and everything else is removed to some other helper class or helper function.